### PR TITLE
Prove ensymfib

### PR DESCRIPTION
**Pull request by BTernaryTau**
f1oenfi

> If the domain of a one-to-one, onto function is finite, then the domain and range of the function are equinumerous.  This theorem is proved without using the Axiom of Replacement or the Axiom of Power Sets (unlike ~ f1oeng ).

f1oenfirn

> If the range of a one-to-one, onto function is finite, then the domain and range of the function are equinumerous.

enreffi

> Equinumerosity is reflexive for finite sets, proved without using the Axiom of Power Sets (unlike ~ enrefg ).

ensymfib

> Symmetry of equinumerosity for finite sets, proved without using the Axiom of Power Sets (unlike ~ ensymb ).

This change also moves [fnfi](https://us.metamath.org/mpeuni/fnfi.html) and specifies that it avoids ax-pow, and it revises [cnvfi](https://us.metamath.org/mpeuni/cnvfi.html) to avoid ax-pow.

This change does not alter axiom usage for any other theorems.

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -89348,6 +89348,83 @@ being the empty set ( ` A e/ _V ` ).  (Contributed by AV, 6-Aug-2019.) $)
...
+    ( cfn wcel cid cres wf1o cen wbr f1oi f1oenfi mpan2 ) ABCAADAEZFAAGHAIAALJK
+    $.
+  $( $j usage 'enreffi' avoids 'ax-pow'; $)
+
+  ${
+    $d A f $.  $d A g $.  $d B f $.  $d B g $.
+    $( Symmetry of equinumerosity for finite sets, proved without using the
+       Axiom of Power Sets (unlike ~ ensymb ).  (Contributed by BTernaryTau,
+       9-Sep-2024.) $)
+    ensymfib $p |- ( A e. Fin -> ( A ~~ B <-> B ~~ A ) ) $=
```
Comment (benjub) [inline set.mm:89420]: Transitivity is next ?

File: set.mm [hunk 2]
```diff
@@ -89348,6 +89348,83 @@ being the empty set ( ` A e/ _V ` ).  (Contributed by AV, 6-Aug-2019.) $)
...
+     ~ f1oeng ).  (Contributed by BTernaryTau, 8-Sep-2024.) $)
+  f1oenfi $p |- ( ( A e. Fin /\ F : A -1-1-onto-> B ) -> A ~~ B ) $=
+    ( cfn wcel wf1o cen wbr wfn f1ofn fnfi sylan ancoms f1oen3g sylancom ) ADEZ
+    ABCFZCDEZABGHQPRQCAIPRABCJACKLMABCDNO $.
+  $( $j usage 'f1oenfi' avoids 'ax-pow' 'ax-rep'; $)
+
+  $( If the range of a one-to-one, onto function is finite, then the domain and
+     range of the function are equinumerous.  (Contributed by BTernaryTau,
+     9-Sep-2024.) $)
+  f1oenfirn $p |- ( ( B e. Fin /\ F : A -1-1-onto-> B ) -> A ~~ B ) $=
```
Comment (benjub) [inline set.mm:89400]: There is a possible third statement with antecedent `F e. Fin`.

File: set.mm [hunk 3]
```diff
@@ -90665,9 +90728,10 @@ Axiom of Infinity (see comments in ~ fin2inf ).  (Contributed by NM,
     ( cid cres cfn wcel cdm dmfi eqeltrrid wfn wfun funi funfn mpbi resfnfinfin
     dmresi mpan impbii ) BACZDEZADEZSARFDAORGHBBFZIZTSBJUBKBLMUAABNPQ $.
 
-  $( If a set is finite, its converse is as well.  (Contributed by Mario
-     Carneiro, 28-Dec-2014.) $)
-  cnvfi $p |- ( A e. Fin -> `' A e. Fin ) $=
+  $( Obsolete version of ~ cnvfi as of 9-Sep-2024.  (Contributed by Mario
+     Carneiro, 28-Dec-2014.)  (Proof modification is discouraged.)
+     (New usage is discouraged.) $)
+  cnvfiOLD $p |- ( A e. Fin -> `' A e. Fin ) $=
```
Comment (benjub) [inline set.mm:90734]:
```suggestion
  $( Alternate proof of ~ cnvfi , shorter but uses ~ ax-pow .  (Contributed by Mario
     Carneiro, 28-Dec-2014.)  (Proof modification is discouraged.)
     (New usage is discouraged.) $)
  cnvfiALT $p |- ( A e. Fin -> `' A e. Fin ) $=
```

File: set.mm (hunk 2 above)
Comment (BTernaryTau) [reply set.mm:89400]: True, but that's already covered by [f1oen3g](https://us.metamath.org/mpeuni/f1oen3g.html), and I don't think this case is important enough to warrant its own theorem.

File: set.mm (hunk 1 above)
Comment (BTernaryTau) [reply set.mm:89420]: Yep!

**Review by icecream17 — APPROVED** [review]

**Review by avekens — APPROVED** [review]

**Pull request merged**